docs/docs/recipes/working-with-images.md

```diff
@@ -51,9 +51,9 @@ export default () => (
 
 ## Reference an image from the `static` folder
 
-As an alternative to importing assets with webpack, the `static` folder allows access to content that gets automatically copied into the `public` folder when built.
+Comme alternative à l'importation d'actifs avec webpack, le dossier `static` permet d'accéder au contenu qui est automatiquement copié dans le dossier `public` une fois construit.
 
-This is an **escape route** for [specific use cases](/docs/static-folder/#when-to-use-the-static-folder), and other methods like [importing with webpack](#import-an-image-into-a-component-with-webpack) are recommended to leverage optimizations made by Gatsby.
+Il s'agit d'une **route d'échappement** pour [cas d'utilisation spécifiques](/docs/dossier-statique/#quand-utiliser-le-dossier-statique), et d'autres méthodes comme [importation avec webpack](#import-an-image-into-a-component-with-webpack) sont recommandés pour tirer parti des optimisations faites par Gatsby.
```

Comment:
Parts of this sentence seems to have been translated separately in Google translate.